### PR TITLE
Document pool_limits and SSL context on verify

### PR DESCRIPTION
**Pull request by yeraydiazdiaz**
Closes https://github.com/encode/httpx/issues/926 and https://github.com/encode/httpx/issues/787

### CODE REVIEW

**Comment by yeraydiazdiaz** [comment]
That flaky test is failing on Python 3.7 now... https://github.com/encode/httpx/runs/666709254

File: docs/advanced.md [hunk 1]
```diff
@@ -399,6 +399,22 @@ client = httpx.Client(timeout=timeout)
...
 
+## Pool limit configuration
+
+You can control the connection pool size using the `pool_limits` keyword
+argument on the client. It takes instances of `httpx.PoolLimits` which define:
+
+- `soft_limit`, number of allowable keep-alive connections, or `None` to always
+allow. (Defaults 10)
+- `hard_limit`, maximum number of allowable connections, or` None` for no limits.
+(Default 100)
```
Comment (tomchristie) [inline docs/advanced.md:410]: This highlights to me that we're using the clearer "max_keepalive" and "max_connections" in `httpcore`.

I wonder if we should deprecate across to that naming for 1.0?

**Comment by tomchristie** [comment]
Helps highlight a possible naming change, but yup good with this.

File: docs/advanced.md (hunk 1 above)
Comment (yeraydiazdiaz) [reply docs/advanced.md:410]: Yeah, I agree it's clearer.

**Pull request merged**